index.html

```diff
@@ -0,0 +1,253 @@
+              </li>
+              <li class="nav-item">
+                <a class="nav-link mr-2" href="#">
+                  <i class="fa fa-twitter text-white"></i>
+                </a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link text-white" href="#">
+                  English
+                </a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link text-white" href="#">
+                  My Page
+                </a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link text-white" href="#">
+                  Logout
+                </a>
+              </li>
+            </ul>
+        </nav>
+        <nav class="navbar navbar-expand-sm bg-white">
+            <a class="navbar-brand p-0 m-0" href="index.html">   
+                <img class="img-fluid" src="assets/images/logo.png" alt="Logo">
+            </a>
+            <ul class="navbar-nav ml-auto">
+              <li class="nav-item">
+                <a class="nav-link pr-5" href="index.html">
+                  Home
+                </a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link pr-5" href="about.html">
+                  About
+                </a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link pr-5" href="#">
+                  Join
+                </a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link pr-5" href="#">
+                  Sponsor
+                </a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link pr-5" href="#">
+                  News
+                </a>
+              </li>
+              <li class="nav-item pr-4 pl-4 campaign">
+                <a class="nav-link orange pr-5" href="#">
+                  CM Campaign
+                </a>
+              </li>
+            </ul>
+        </nav>
+        </div>
+        <nav class="toggle d-md-none d-xs-block">
+        <img src="assets/images/menu.png" alt="hamburger menu" class="hamburger_menu_icon" />
+        <ul class="mobile_side_nav">
+          <li class="menu_link closebtn"><a href="javascript:void(0)">&times;</a></li>
+          <li class="menu_link"><a href="index.html" class="menulink">Home</a><hr class="section_title_hr"/></li>
+          <li class="menu_link"><a href="about.html" class="menulink">About</a><hr class="section_title_hr"/></li>
+          <li class="menu_link"><a href="#">Join</a><hr class="section_title_hr"/></li>
+          <li class="menu_link"><a href="#">Sponsor</a><hr class="section_title_hr"/></li>
+          <li class="menu_link"><a href="#">News</a><hr class="section_title_hr"/></li>
+        </ul>
+        </nav>
+    </header>
+
+    <main class="parent_main">
+       <section class="container-fluid jumbotron slider intro_header pr-5">
```

Comment:
- [ ]  You implemented the footer and the partners' section in a good way 👏. Let's make it even greater by a margin on the left so your UI design is better.
- Your design:
<img width="950" alt="image" src="https://user-images.githubusercontent.com/77190203/157694069-51f43ef2-b8e0-4b5a-8960-1b5be0e29e6a.png">

- Original design:
<img width="626" alt="image" src="https://user-images.githubusercontent.com/77190203/157694977-f443d766-e928-49ff-b877-a86488a676fc.png">